types/base.d.ts

```diff
@@ -1,10 +1,26 @@
-import { OrderAPI } from "./order";
-import { ReviewAPI } from "./review";
+export type BuildOptions = {
+    headers?: Record<string, unknown>;
+    kwargs?: Record<string, unknown>;
+    params?: Record<string, unknown>;
+    handle?: boolean;
+    data?: object,
+    dataJ?: JSON,
+    dataM?: object,
+}
 
-export interface API extends OrderAPI, ReviewAPI {}
+export enum BuildMethod {
+    GET = "GET",
+    POST = "POST",
+    PUT = "PUT",
+    DELETE = "DELETE",
+    PATCH = "PATCH",
+    OPTIONS = "OPTIONS"
+}
 
 export declare class API implements API {
     constructor(kwargs?: object);
     load(): Promise<void>;
+    build(method: BuildMethod, url: string, options?: BuildOptions);

```

Comment:
No need for this one, it's inherited from `yonius.API`. Speaking of which, that's not being done in the type